scripts/update_robots.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+#------------------------------------------------------------------------------
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) 2021 solidity contributors.
+#------------------------------------------------------------------------------
+
+set -eu
+
+repo_root="$(dirname "$0")/.."
```

Comment:
```suggestion
repo_root=`dirname "$0"`/..
```

Reply:
Well, that will cause a shellcheck warning :)
The `$()` form is generally recommended over ``` `` ``` these days.

Reply:
@cameel that is interesting. I am pretty sure I know why shellcheck complains (it's easier to accidentally do mistakes), but using back ticks avoids spawning a full sub shell. :)

Reply:
Hmm... I never heard of that difference so I tried to verify but for me it appears that both run in a subshell.

When I run this on my machine, both have a different PID than the main shell:
```bash
echo "$(echo $BASHPID) vs $BASHPID"
echo "`echo $BASHPID` vs $BASHPID"
```
```
119218 vs 117835
119219 vs 117835
```

Anyway, most of our scripts are not performance-critical so this does not really matter :) For me the biggest advantage of `$()` is that it can be nested in a sane way and does not require weird escaping of quotes and slashes.